### ISSUE TITLE
[Bug][DemoApp][Android] The native tab-bar is displayed on top of the customize bottom sheet

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#358]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S22

### OS Version

Android 14

### App version

0.6.0

### Describe the issue

The OS native tab-bar is displayed on top of the customize bottom sheet
<img width="570" height="259" alt="" src="https://github.com/user-attachments/assets/86383080-8ada-475a-9037-dff2d7bd3901" />

### Expected Behavior

Expected display:
<img width="467" height="179" alt="" src="https://github.com/user-attachments/assets/6ae7d73c-9964-4231-9f81-f2325abaaf08" />

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 444
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 444
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
